src/Momento.Sdk/Incubating/SimpleCacheClient.cs

```diff
@@ -639,23 +639,23 @@ public async Task<CacheListFetchResponse> ListFetchAsync(string cacheName, strin
     /// <param name="value">The value to completely remove from the list.</param>
     /// <returns>Task representing the result of the cache operation.</returns>
     /// <exception cref="ArgumentNullException">Any of <paramref name="cacheName"/>, <paramref name="listName"/>, or <paramref name="value"/> is <see langword="null"/>.</exception>
-    public async Task<CacheListRemoveAllResponse> ListRemoveAllAsync(string cacheName, string listName, byte[] value)
+    public async Task<CacheListRemoveAllResponse> ListRemoveValueAsync(string cacheName, string listName, byte[] value)
     {
         Utils.ArgumentNotNull(cacheName, nameof(cacheName));
         Utils.ArgumentNotNull(listName, nameof(listName));
         Utils.ArgumentNotNull(value, nameof(value));
 
-        return await this.dataClient.ListRemoveAllAsync(cacheName, listName, value);
+        return await this.dataClient.ListRemoveValueAsync(cacheName, listName, value);
     }
 
-    /// <inheritdoc cref="ListRemoveAllAsync(string, string, byte[])"/>
-    public async Task<CacheListRemoveAllResponse> ListRemoveAllAsync(string cacheName, string listName, string value)
+    /// <inheritdoc cref="ListRemoveValueAsync(string, string, byte[])"/>
+    public async Task<CacheListRemoveAllResponse> ListRemoveValueAsync(string cacheName, string listName, string value)
```

Comment:
just thinking out loud but I feel like inevitably someone is going to want a `RemoveValueOnce` vs `RemoveValueAll` ik just doing a rename but the new name even more so makes me think there should be a way to remove just one element in the list of a certain type.

Reply:
The way this is named now I would sort of expect this to just remove 1 element instead of all and then maybe for us to add a `filter` function to list as well?

Reply:
We will provide the options `ListRemoveValueFront(value, count)` to mean "remove the first `count` occurrences" and `ListRemoveValueBack(value, count)` to mean "remove the last `count` occurrences.